### DISCUSSION TITLE
3.5.1版本无法播放别人分享的阿里云盘中的视频，怎么办

### DISCUSSION THREAD

**Discussion by skyyezhu** [AlistGo/alist#2563] [Q&A]
各位大神，我的ALIST是3.5.1，挂载别人分享的阿里云网盘后可以正常访问文件，但无法播放视频。尝试打开或关闭WEB代理，也不行。WEBDAV重定向选择302重定向、本地代理，都无法播放视频。咋办？

**Comment by iemobiu** [comment]
这个我也遇到了，经过一些测试，发现是视频编码的问题，有的视频是可以正常播放的，同时还挑浏览器。
感觉是不是alist在视频解码这块能够增强一些。没用过别的版本，从V3.5.1开始用的。

**Comment by skyyezhu** [comment]
应该不是，我把视频转存到我自己的阿里云盘是可以正常播放的，直接挂载别人分享的阿里云盘则无法播放。

**Reply by nxjonson** [reply]
token要用自己的

**Reply by skyyezhu** [reply]
果然，没注意到，自己的TOKEN变了，感谢提醒

**Reply by bigbigking** [reply]
很奇怪的情况。我也是这样，
我的ALIST是3.5.1，挂载别人分享的阿里云网盘后可以正常访问文件，但无法播放视频。尝试打开或关闭WEB代理，也不行。WEBDAV重定向选择302重定向、本地代理，都无法播放视频。
我在AList上添加了2个存储，一个是选的阿里云网盘的驱动，添加的是自己的网盘，用的refreh_token会固定，不会在每次打开这个存储的管理页面的时候变。
另一个存储选的驱动是阿里云盘分享，添加的是上面自己网盘的refreh_token，但是这个保存完下次打开就变了，每次打开都会变。结果就是出现这个issue的状况，能正常浏览文件，但是无法播放视频。
求大佬指点。

**Reply by yjqvgs** [reply]
我也是一模一样的问题，就是无法解决。

**Reply by MYTVBOX** [reply]
换成自己的token一样不行

**Comment by yjqvgs** [comment]
用如下方法获得token，问题解决。
https://alist.nn.ci/zh/guide/drivers/aliyundrive.html
